command/agent/config_test.go

```diff
@@ -266,6 +266,12 @@ func TestConfig_LoadConfig(t *testing.T) {
 		t.Fatalf("bad: %#v", config)
 	}
 
+	expectedConfigFiles := []string{fh.Name()}
+	if !reflect.DeepEqual(config.Files, expectedConfigFiles) {
+		t.Errorf("Loaded configs don't match\nExpected\n%+vGot\n%+v\n",
```

Comment:
I use this pattern when dumping a complex type (struct, list, etc.) because each will appear on its own line. This makes it easier to spot the differences and/or copy-paste into a diff program if it's very large.

For example, this is easy to read:

```
Loaded configs don't match: want 1; got 2
```

But this is not:

```
Loaded configs don't match: want []string{"charlie", "bracvo", "delta"}; got []string{"charlie", "bravco", "delta"}
```

The pattern I used shows up like this when the comparison fails:

```
Loaded configs don't match
Expected
[charlie bracvo delta]
Got
[charlie bravco delta]
```